modules/drivers/bigquery-cloud-sdk/test/metabase/test/data/bigquery_cloud_sdk.clj

```diff
@@ -27,7 +28,15 @@
 
 (sql.tx/add-test-extensions! :bigquery-cloud-sdk)
 
-(defonce ^:private ns-load-time (System/currentTimeMillis))
+(defonce ^:private ^{:arglists '(^java.lang.Long [])} ^{:doc "Timestamp to use for unique dataset identifiers. Initially
```

Comment:
When I was spending like 8 hours straight on fixing BigQuery stuff I was getting annoyed that I kept having to jump to this namespace and redef `ns-load-time` to be able to recreate ta dataset. Changed it so this identifier updates automatically every two hours, so you can reload data without having to reevaluate anything.